File: Modul-4/C/PRAK401-2310817110006-Adrian_Bintang_Saputera.c
#include <stdio.h>

int main()
{
  int angkaKelipatan;
  char simbol;

  printf("Input\n");
  scanf("%d %c", &angkaKelipatan, &simbol);

  printf("Output\n");
  for (int i = 1; i <= 50; i++)
  {
    if (i % angkaKelipatan == 0)
    {
      printf("%c ", simbol);
    }
    else
    {
      printf("%d ", i);
    }
  }

  return 0;
}

File: Modul-4/Python/PRAK401-2310817110006-Adrian_Bintang_Saputera.py
angkaKelipatan, simbol = input("Input\n").split()

angkaKelipatan = int(angkaKelipatan)

print("Output")
for i in range(1, 51):
    if i % angkaKelipatan == 0:
        print(simbol, end=" ")
    else:
        print(i, end=" ")